config/opensearch_dashboards.yml

```diff
@@ -131,8 +131,9 @@
 # Set the allowlist to check input graphite Url. Allowlist is the default check list.
 #vis_type_timeline.graphiteAllowedUrls: ['https://www.hostedgraphite.com/UID/ACCESS_KEY/graphite']
 
-# Set the blocklist to check input graphite Url. Blocklist is an IP list.
+# Set the denylist to check input graphite Url. Denylist is an IP list.
 # Below is an example for reference
+# /** @deprecated use graphiteDeniedIPs */
```

Comment:
I don't think this will show, I think we should just have the yaml file be the updated version of it.